src/target/drivers/usb/devo_usb.c

```diff
@@ -0,0 +1,76 @@
+#include <libopencm3/stm32/rcc.h>
+#include <libopencm3/stm32/gpio.h>
+#include <libopencm3/cm3/nvic.h>
+#include <libopencm3/usb/usbd.h>
+#include <libopencm3/usb/msc.h>
+
+#include "common.h"
+#include "target/drivers/mcu/stm32/rcc.h"
+
+#include "devo_usb.h"
+
+usbd_device *usbd_dev;
+
+uint8_t usbd_control_buffer[128];
+
+const struct usb_device_descriptor dev_descr = {
+    .bLength = USB_DT_DEVICE_SIZE,
+    .bDescriptorType = USB_DT_DEVICE,
+    .bcdUSB = 0x0200,
+    .bDeviceClass = 0,
+    .bDeviceSubClass = 0,
+    .bDeviceProtocol = 0,
+    .bMaxPacketSize0 = 64,
+    .idVendor = 0x0483,
+    .idProduct = 0x5720,
+    .bcdDevice = 0x0200,
+    .iManufacturer = 1,
+    .iProduct = 2,
+    .iSerialNumber = 3,
+    .bNumConfigurations = 1,
+};
+
+const char USB_Product_Name[] = "DeviationTx";
+const char USB_Serial_Number[] = "5.0";
+
```

Comment:
I don't like hardcoding this version here.  I'll never remember to change it.  can we use the version-string instead?

Reply:
fixed